db.go

```diff
@@ -227,6 +228,10 @@ func newDBMetrics(db *DB, r prometheus.Registerer) *dbMetrics {
 		Name: "prometheus_tsdb_storage_blocks_bytes",
 		Help: "The number of bytes that are currently used for local storage by all blocks.",
 	})
+	m.sizeRetention = prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "prometheus_tsdb_retention_byte_limit",
+		Help: "Max number of bytes to be retained in the tsdb blocks, 0 or less means disabled",
```

Comment:
0 means disabled.

Let's not make things harder for users than we have to. I'd also put the word configured in there.

Reply:
@brian-brazil Updated.